charms/knative-serving/src/charm_template.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
I believe that we already have enough different implementations of doing this in different charm repositories, if we want to go ahead with this approach I think it is time for us to start with an initial implementation of the lightkube library.